app/controllers/participants/index.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    // Manually specify column format options for model table
+    columns: [
+        {propertyName: 'id', title:'ID'},
+        {propertyName: 'firstName'},
+        {propertyName: 'lastName'},
+        {propertyName: 'modifiedOn', title: 'Last active'},  // TODO: That's not quite what this field does, but useful first approximation
+        {title: '', template: 'custom/profile-link'}
```

Comment:
@abought I know this is already merged, but in the process of reviewing https://github.com/CenterForOpenScience/experimenter/pull/17/, some questions are arising for me.

Is the `custom/` prefix a requirement for this addon? 

I can see some value in putting our custom table-cell templates in a common folder, but maybe the folder name could be more descriptive? I know this is seems nit-picky, but 'custom' can have a lot of meanings.